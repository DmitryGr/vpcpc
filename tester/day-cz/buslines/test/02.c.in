100
33 57
7 33
95 33
12 33
33 75
33 54
50 33
33 11
38 33
41 33
34 33
48 33
77 33
33 99
55 33
62 33
53 33
4 33
1 33
33 83
33 78
56 33
33 82
52 33
33 71
86 33
33 84
79 33
33 14
33 36
33 73
80 33
33 23
33 2
33 26
47 33
32 33
33 85
93 33
17 33
25 33
33 51
33 65
33 92
24 33
67 33
33 59
3 33
46 33
97 33
33 66
96 33
100 33
72 33
33 74
13 33
58 33
33 8
33 87
98 33
6 33
42 33
44 33
31 33
33 10
16 33
33 94
33 61
89 33
88 33
45 33
37 33
33 19
33 21
33 28
76 33
33 68
33 69
33 49
64 33
33 39
33 90
33 29
91 33
18 33
20 33
33 40
5 33
70 33
27 33
35 33
33 81
63 33
33 9
33 22
33 60
30 33
33 15
43 33
