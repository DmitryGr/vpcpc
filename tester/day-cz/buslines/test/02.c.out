99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
4950
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
99
